yb-voyager/versions/ci-config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "versions": {
+    "go": "1.23.1",
+    "java": "17",
+    "staticcheck": "2025.1.1"
+  },
+  "runner": {
+    "ubuntu": "ubuntu-22.04"
```

Comment:
i think runner here is not used by workflows right now
Will take it up along with review comments.

Reply:
remove this for now, seems like a overkill for ubuntu versioning